codecparsers/h265Parser.cpp

```diff
@@ -318,12 +347,12 @@ bool Parser::profileTierLevel(ProfileTierLevel* ptl, NalReader& nr,
                 || ptl->sub_layer_profile_compatibility_flag[4]
                 || ptl->sub_layer_profile_compatibility_flag[5])
                 //The number of bits in this syntax structure is not affected by this condition
-                    ptl->sub_layer_inbld_flag[i] = nr.read(1);
+				READ(nr, ptl->sub_layer_inbld_flag[i]);
             else
                 nr.skip(1); // sub_layer_reserved_zero_bit[i]
```

Comment:
@xuguangxin  if that is the case, i think h264 also need to do some chages, because i found lines such as br.skip(1) in the h264 file.